spec/javascripts/packages/document-capture/components/document-capture-spec.jsx

```diff
@@ -514,6 +514,10 @@ describe('document-capture/components/document-capture', () => {
             json: () => ({ success: false, remaining_attempts: 1, errors: [{}] }),
           });
 
+        expect(queryByText('idv.troubleshooting.options.verify_in_person')).not.to.exist();
+        await userEvent.click(getByText('forms.buttons.submit.default'));
+        expect(queryByText('idv.troubleshooting.options.verify_in_person')).not.to.exist();
```

Comment:
> Test expanded to capture bug

I'd contemplated not including any spec coverage here, since the way it had been implemented was as as additive feature of the documents step, and ideally I'd be removing whichever spec had asserted its presence, but one did not exist. But with all the different conditions involved in rendering troubleshooting options, I figured it wouldn't hurt to check that it's not visible when we're not expecting it to be.